composer.json

```diff
@@ -43,10 +43,7 @@
         }
     },
     "config": {
-        "sort-packages": true,
-        "platform": {
-            "php": "7.1.0"
-        }
```

Comment:
It will be hard for you to produce a good lock file if you remove this.

Reply:
Why is there a lock file?

Reply:
Also https://github.com/doctrine/cache doesn't have this in its composer.json file either.

Reply:
It's a policy that is enforced in some but not all repositories to avoid issues with external tools such as phpcs, phpstan, Psalm etc.